### PR TITLE
feat: the point -no political use - added

### LINKED ISSUE #2750: 🚀 [Feature] terms of use extend point - no parties
> [ogerly](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-13T15:18:34Z" title="Monday, January 13th 2020, 4:18:34 pm +01:00">Jan 13, 2020</time>_
_Closed <time datetime="2020-02-22T15:53:27Z" title="Saturday, February 22nd 2020, 4:53:27 pm +01:00">Feb 22, 2020</time>_
---

## :rocket: Feature
terms of use extend point - no parties 

it concerns only party in the political sense. 

no organizations

### PR DESCRIPTION
**Pull request by Tirokk**
> [ogerly](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-02-22T13:50:01Z" title="Saturday, February 22nd 2020, 2:50:01 pm +01:00">Feb 22, 2020</time>_
_Merged <time datetime="2020-02-22T15:53:26Z" title="Saturday, February 22nd 2020, 4:53:26 pm +01:00">Feb 22, 2020</time>_
---

## 🍰 Pullrequest
the terms of use are extended by one point. 

> No Parties

see picture:

the version number was increased from 0.0.3 to **0.0.4.** 

FireShot Capture 330 - Terms and conditions - Human Connection - localhost

### Issues
 - fixes #2750

### CODE REVIEW

**Comment by Tirokk** [comment]
> [cypress[bot]](https://github.com/apps/cypress) **Authored by [cypress[bot]](https://github.com/apps/cypress)**
_<time datetime="2020-02-22T14:16:12Z" title="Saturday, February 22nd 2020, 3:16:12 pm +01:00">Feb 22, 2020</time>_

---



<p align="center">
  <br />
  
</p>

---

### Test summary
<a href="https://dashboard.cypress.io/projects/qa7fe2/runs/5524/specs"></a> <b title="Passed">66</b>  &bull; <a href="https://dashboard.cypress.io/projects/qa7fe2/runs/5524/failures"></a> <b title="Failed">0</b>  &bull;  <b title="Pending">0</b> &bull;  <b title="Skipped">0</b>

---

### Run details
| | |
| --- | --- |
| **Project** | Human-Connection |
| **Status** | Passed |
| **Commit** | 6472cec0c6 |
| **Started** | Feb 22, 2020 1:56 PM |
| **Ended** | Feb 22, 2020 2:16 PM |
| **Duration** | 19:52 <a href="https://on.cypress.io/parallelization" title="Enable parallelization to reduce run duration">💡</a> |
| **OS** | Linux Ubuntu Linux - 16.04 |
| **Browser** | Firefox 68  |

[View run in Cypress Dashboard ➡️](https://dashboard.cypress.io/projects/qa7fe2/runs/5524/specs)



---

This comment has been generated by **cypress-bot** as a result of this project's GitHub integration settings. You can manage this integration in [this project's settings in the Cypress Dashboard](https://dashboard.cypress.io/projects/qa7fe2/settings)

**Comment by Tirokk** [comment]
> [cypress[bot]](https://github.com/apps/cypress) **Authored by [cypress[bot]](https://github.com/apps/cypress)**
_<time datetime="2020-02-22T14:28:56Z" title="Saturday, February 22nd 2020, 3:28:56 pm +01:00">Feb 22, 2020</time>_

---



<p align="center">
  <br />
  
</p>

---

### Test summary
<a href="https://dashboard.cypress.io/projects/qa7fe2/runs/5525/specs"></a> <b title="Passed">66</b>  &bull; <a href="https://dashboard.cypress.io/projects/qa7fe2/runs/5525/failures"></a> <b title="Failed">0</b>  &bull;  <b title="Pending">0</b> &bull;  <b title="Skipped">0</b>

---

### Run details
| | |
| --- | --- |
| **Project** | Human-Connection |
| **Status** | Passed |
| **Commit** | cad47808b3 <span style="cursor:help;" title="Your CI provider generated this commit by merging this branch into the target branch.">ℹ️</span> |
| **Started** | Feb 22, 2020 2:08 PM |
| **Ended** | Feb 22, 2020 2:28 PM |
| **Duration** | 20:53 <a href="https://on.cypress.io/parallelization" title="Enable parallelization to reduce run duration">💡</a> |
| **OS** | Linux Ubuntu Linux - 16.04 |
| **Browser** | Firefox 68  |

[View run in Cypress Dashboard ➡️](https://dashboard.cypress.io/projects/qa7fe2/runs/5525/specs)



---

This comment has been generated by **cypress-bot** as a result of this project's GitHub integration settings. You can manage this integration in [this project's settings in the Cypress Dashboard](https://dashboard.cypress.io/projects/qa7fe2/settings)

File: cypress/constants/terms-and-conditions-version.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
 // please change also version in file "webapp/constants/terms-and-conditions-version.js"
-export const VERSION = '0.0.3'
\ No newline at end of file
+export const VERSION = '0.0.4'
```
Comment (Tirokk) [inline cypress/constants/terms-and-conditions-version.js:2]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-22T15:53:06Z" title="Saturday, February 22nd 2020, 4:53:06 pm +01:00">Feb 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,2 +1,2 @@
 // please change also version in file "webapp/constants/terms-and-conditions-version.js"
-export const VERSION = '0.0.3'
\ No newline at end of file
+export const VERSION = '0.0.4'
```


Nice! :+1:

**Review by Mogge — APPROVED** [review]
Good :clap:

**Pull request closed without merging**